examples/helloworld/lib/helloworld.pb.ex

```diff
@@ -1,31 +1,25 @@
 defmodule Helloworld.HelloRequest do
-  use Protobuf
+  @moduledoc false
+  use Protobuf, protoc_gen_elixir_version: "0.10.0", syntax: :proto3
```

Comment:
Did you add this manually or did protoc generate this via the protobuf plugin? I don't think it's cool that we have to change these files after they've been created if that's the case here

Reply:
@sleipnir i generate it with protoc. This change added since plugin version 0.10